airflow/www/security.py

```diff
@@ -484,7 +484,7 @@ def create_dag_specific_permissions(self, session=None):
             dag_id = row[0]
             for perm_name in self.DAG_PERMS:
                 dag_resource_name = self.prefixed_dag_id(dag_id)
-                if dag_resource_name and perm_name and (dag_resource_name, perm_name) not in perms:
+                if (perm_name, dag_resource_name) not in perms:
```

Comment:
Checking that we get a dag_id (which is the PK) and that it gets prefixed properly didn't seem necessary, unless I've overlooked something?